dynamixel_workbench_toolbox/src/dynamixel_workbench/dynamixel_driver.cpp

```diff
@@ -594,6 +583,12 @@ const char *DynamixelDriver::findModelName(uint16_t model_num)
     model_name = "PRO-H54-100-S500-R";
   else if (num == PRO_H54_200_S500_R)
     model_name = "PRO-H54-200-S500-R";
+  else if (num == BITFOOT)
+    model_name = "BITFOOT";
+  else if (num == BITBOTS_IMU_MODULE)
+    model_name = "IMU";
+  else if (num == BITBOTS_CORE)
+    model_name = "CORE";
 
```

Comment:
This whole block should be refactored, but works for now.

Reply:
follows the robotis convention in this file. did not want to change complete files to our code style